### DISCUSSION TITLE
Why allkernel option is not in README or help message?

### DISCUSSION THREAD

**Discussion by Winson-Huang** [async-profiler/async-profiler#810] [Q&A] [answered]
In argument.cpp there is a allkernel JVMTI argument, but it's not in README or help message of profiler.sh, what's the reason?

**Comment by apangin** [comment, marked as answer]
There is no 1:1 mapping between agent arguments and launcher options, and never meant to be. `allkernel` option is not supported in the launcher and therefore not listed in the help message or README. It might be also removed from future versions of async-profiler, as there  was hardly any use for it. If you disagree with that, feel free to open an issue and describe your use case.
